great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -153,12 +153,17 @@ def __init__(
 
         self._usage_statistics_handler = usage_statistics_handler
 
-        # Necessary to annotate ExpectationSuite during `run()`
+        # Necessary to annotation ExpectationSuite during `run()`
+        rule_to_load_into_citation: dict = {}
+        if isinstance(rules, Rule):
+            rule_to_load_into_citation = rules.to_json_dict()
```

Comment:
@Shinnnyshinshin This is not only applying the singular `Rule` type to the plural `rules` list, but is also not needed, because `rules: Optional[Dict[str, Dict[str, Any]]]` (i.e., it is already a dictionary).  Thanks.

Reply:
Also unnecessary after the changes to `add_rule()`